compcheck/secretbinding_check.go

```diff
@@ -728,7 +728,11 @@ func VerifyVaultSecrets(secretBinding []exchangecommon.SecretBinding, nodeOrg st
 			if exists, err := VerifySingleVaultSecret(vaultSecretName, nodeOrg, agbotURL, vaultSecretExists, msgPrinter); err != nil {
 				ret[vaultSecretName] = err.Error()
 			} else if !exists {
-				ret[vaultSecretName] = msgPrinter.Sprintf("Secret %v does not exist in the secret manager.", vaultSecretName)
+				msg := msgPrinter.Sprintf("Secret %v does not exist in the secret manager.", vaultSecretName)
+				if sn.EnableNodeLevelSecrets {
+					msg = msgPrinter.Sprintf("Secret %v does not exist in the secret manager for neither org level or user level.", vaultSecretName)
```

Comment:
I think change `neither` to `either` would be more correct so we have either ... and or